tests/test_command_repo_files.py

```diff
@@ -0,0 +1,149 @@
+import unittest
+from argparse import ArgumentParser
+from unittest.mock import Mock, patch
+
+from huggingface_hub.commands.repo_files import DeleteSubCommand, RepoFilesCommand
+
+from .testing_utils import DUMMY_MODEL_ID
+
+
+class TestRepoFilesCommand(unittest.TestCase):
```

Comment:
Can you move this to `tests/test_cli.py` please? Keep it as a new test class as you did but not in a new module.